toc_config.py

```diff
@@ -395,15 +395,31 @@ def parse_chapter(docname, doc, parent, module_meta):
 
     # Create categories.
     category_names = app.config.category_names
+    for key, name in category_names.items():
+        category = app.config.categories.get(key, {"name": name})
+        if name != category.get("name", name):
+            raise SphinxError(f"Name mismatch between config options 'category_names' and 'categories' for key '{key}': '{name}' vs '{category['name']}'")
+
     categories = {
         key: {
             'name': category_names.get(key, key),
         } for key in category_keys
     }
+
+    for key, category in app.config.categories.items():
+        if key not in categories:
+            raise SphinxError(f"Category with key '{key}' was configured but not used in the material")
+
+        categories[key].update(category)
```

Comment:
Should this part be moved after the following `for` loop that forces `chapter` and `feedback` categories to `status` `nototal`? If somebody really wanted to set a different status, then they could do it with the new `categories` setting.

Some courses actually want to show feedback points to the students and changing the status of the feedback category would be then useful.